HorizonTechnicalDoc.md

```diff
@@ -7297,4 +7301,4 @@ In the table below:
 | [OnProjectileHitObject](#projectile-launcher-gizmo) | `objectHit: Entity`<br/>`position: Vec3, normal: Vec3` |
 | [OnProjectileHitPlayer](#projectile-launcher-gizmo) | `playerHit: Player`<br/>`position: Vec3, normal: Vec3, headshot: boolean` |
 | [OnProjectileHitWorld](#projectile-launcher-gizmo) | `position: Vec3`<br/>`normal: Vec3` |
-| [OnProjectileLaunched](#projectile-launcher-gizmo) | `launcher: Entity` |
\ No newline at end of file
+| [OnProjectileLaunched](#projectile-launcher-gizmo) | `launcher: Entity` |
```

Comment:
not sure what's different on this line. Some whitespace?  Didn't intentionally touch it.